CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+### 0.0.1: First Release
+
+ - First release published to npm
+
+### 0.0.2: Second Release
+
+**Enhancements**
+
+**Fixes**
+
```

Comment:
```suggestion
### v0.1.0 (07-8-21)
 - Initial development release
```
- First release should be v0.1.0, not 0.0.1.
- It's a good idea to include the release date.
- We should specify in the description that it's a development release, since "first release" is ambiguous as to whether or not it's stable or unstable.
- We don't want to preemptively include the next release, since it could be "v0.1.1", "v0.2.0", or even "v1.0.0", depending on what's added/changed. The next release might only include enhancements and no fixes, or vice versa.

Reply:
That makes sense, I have never used a `CHANGELOG` file so I wanted to get some advices and further edit it